src/backend/marsha/core/tests/test_views_lti_select.py

```diff
@@ -352,6 +354,8 @@ def test_views_lti_select_behind_tls_termination_proxy(self):
         )
         context = json.loads(unescape(match.group(1)))
 
+        print(context.get("videos"))
+
```

Comment:
```suggestion
```

Reply:
![](https://i.redd.it/uj5aput23d281.jpg)